site/layouts/partials/footer.html

```diff
@@ -1,59 +1,78 @@
+          <li class="mb-3"><a href="{{ .Site.Params.github_org }}/bootstrap" target="_blank" rel="noopener" class="text-decoration-none text-body-secondary hover-primary">Bootstrap 5</a></li>
+          <li class="mb-3"><a href="{{ .Site.Params.github_org }}/bootstrap/tree/v4-dev" target="_blank" rel="noopener" class="text-decoration-none text-body-secondary hover-primary">Bootstrap 4</a></li>
+          <li class="mb-3"><a href="{{ .Site.Params.github_org }}/icons" target="_blank" rel="noopener" class="text-decoration-none text-body-secondary hover-primary">Icons</a></li>
+          <li class="mb-3"><a href="{{ .Site.Params.github_org }}/rfs" target="_blank" rel="noopener" class="text-decoration-none text-body-secondary hover-primary">RFS</a></li>
+          <li class="mb-3"><a href="{{ .Site.Params.github_org }}/examples" target="_blank" rel="noopener" class="text-decoration-none text-body-secondary hover-primary">Examples repo</a></li>
         </ul>
       </div>
-      <div class="col-6 col-lg-2 mb-3">
-        <h5>Community</h5>
+
+      <!-- Community Column -->
+      <div class="col-6 col-lg-2">
+        <h5 class="fw-semibold mb-4">Community</h5>
         <ul class="list-unstyled">
-          <li class="mb-2"><a href="{{ .Site.Params.github_org }}/bootstrap/issues" target="_blank" rel="noopener">Issues</a></li>
-          <li class="mb-2"><a href="{{ .Site.Params.github_org }}/bootstrap/discussions" target="_blank" rel="noopener">Discussions</a></li>
-          <li class="mb-2"><a href="https://github.com/sponsors/twbs" target="_blank" rel="noopener">Corporate sponsors</a></li>
-          <li class="mb-2"><a href="{{ .Site.Params.opencollective }}" target="_blank" rel="noopener">Open Collective</a></li>
-          <li class="mb-2"><a href="https://stackoverflow.com/questions/tagged/bootstrap-5" target="_blank" rel="noopener">Stack Overflow</a></li>
+          <li class="mb-3"><a href="{{ .Site.Params.github_org }}/bootstrap/issues" target="_blank" rel="noopener" class="text-decoration-none text-body-secondary hover-primary">Issues</a></li>
+          <li class="mb-3"><a href="{{ .Site.Params.github_org }}/bootstrap/discussions" target="_blank" rel="noopener" class="text-decoration-none text-body-secondary hover-primary">Discussions</a></li>
+          <li class="mb-3"><a href="https://github.com/sponsors/twbs" target="_blank" rel="noopener" class="text-decoration-none text-body-secondary hover-primary">Corporate sponsors</a></li>
+          <li class="mb-3"><a href="{{ .Site.Params.opencollective }}" target="_blank" rel="noopener" class="text-decoration-none text-body-secondary hover-primary">Open Collective</a></li>
+          <li class="mb-3"><a href="https://stackoverflow.com/questions/tagged/bootstrap-5" target="_blank" rel="noopener" class="text-decoration-none text-body-secondary hover-primary">Stack Overflow</a></li>
```

Comment:
For the links style, it should be handled in the corresponding (S)CSS file, not by repeating classes all over the file.